AgileCoffeeShop/Coordinate.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace AgileCoffeeShop
+{
+    public class Coordinate
+    {
+        private double _xCoordinate;
+        private double _yCoordinate; 
+        public double XCoordinate
+        {
+            get
+            {
+                return _xCoordinate;
+            } 
+            set
+            {
+                if(value >=-180 && value <= 180)
+                {
+                    _xCoordinate = value;
+                }
+                else
+                {
+                    Console.WriteLine("Invalid value for x coordinate. Program will exit.\nPress any key to continue...");
```

Comment:
Maybe you can refactor this 2 errors from setting the xCoordinate and yCoordinate? It seems as it's the same string, only x and y being different.

Reply:
Yes, I will do that.